### ISSUE TITLE
Wrong tag 5.4.1

### ISSUE THREAD

**Issue by q-b** [RocketChat/helm-charts#84]
https://github.com/RocketChat/helm-charts/releases/tag/5.4.1

`Chart.yaml` still point to version: *5.3.2*

**Comment by evilscientress** [comment]
Yes the issue is that the [CI Job](https://github.com/RocketChat/helm-charts/actions/runs/3779298231) failed for the 5.4.1 release and it never was deployed to Github pages.

But this is a more widespread issue. All commit CI jobs have been failing since November 21st 2022 and nobody seams to care.
